Source/macOS/LoopbackHTTPServer/OIDLoopbackHTTPServer.h

```diff
@@ -72,6 +72,9 @@ typedef enum {
 - (BOOL)start:(NSError **)error;
 - (BOOL)stop;
 
+- (BOOL)hasIPv4Socket;
```

Comment:
Since `start:` always returns `NO` if it can't create an IPv4 socket, why is this necessary?